### ISSUE TITLE
VB -> C# - String Interpolation With Double Quotes Not Properly Escaping Double Quotes

### ISSUE THREAD

**Issue by knodel12** [icsharpcode/CodeConverter#192]
### Input code
```VB
Public Class Test
    Public Function StringInter(t As String) As String
        Return $"{t} "" t"
    End Function
End Class
```

### Erroneous output
```C#
    public class Test
    {
        public string StringInter(string t)
        {
            return $"{t} "" t"
        }
    }
```

### Expected output
```C#
    public class Test
    {
        public string StringInter(string t)
        {
            return $"{t} \" t"
        }
    }
```

### Details
Product in use: e.g. VS extension

Version in use: 6.1.0

@GrahamTheCoder I'm wondering why any strings with double quotes aren't just converted to string literals.  In this case, an alternative solution could've been to put the @ sign at the start of the string rather than escaping the double quote.  Considering all strings in VB are literals and the logic around escaping a double quote is the same as literals in C#, it seems like any string with a quote should just end up as a C# literal.

**Comment by GrahamTheCoder** [comment]
The relevant code does only a slightly more complicated version of what you've described. But obviously has a bug: https://github.com/icsharpcode/CodeConverter/blob/master/ICSharpCode.CodeConverter/CSharp/CommonConversions.cs#L190

The intent is to convert to a string literal whenever there are characters that otherwise need escaping. Always having the literal `@` felt overkill and looked weird in some contexts (maybe the tests will capture some of those contexts). Looks like I made an exception for quote marks (Maybe I just thought `@""""""` is more confusing than `"\"\""`.)

**Comment by knodel12** [comment]
@GrahamTheCoder Created a pull request.  I believe I linked it properly... Fairly new to contributing to open source projects.

**Comment by GrahamTheCoder** [comment]
Yep all good. For future, if you put "Fixes #192", it works as a link so this issue gets auto-closed when the PR is merged.
